platform/config/config-model-impl/src/main/java/org/codice/ddf/config/model/impl/AbstractConfig.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.config.model.impl;
+
+import java.util.Objects;
+import org.codice.ddf.config.Config;
+
+public abstract class AbstractConfig implements Config {
```

Comment:
Must implement the ConfigSingleton interface and maybe it should be renamed AbstractSingletonConfig

Reply:
Fixed.  I made a lot of changes to the model impl.